{
  "modified": "2024-10-24T01:01:59Z",
  "published": "2024-06-25T13:38:31Z",
  "schema_version": "1.5.0",
  "id": "MAL-2024-5525",
  "aliases": [
    "SNYK-PYTHON-PIPFONTINGV2-5932351"
  ],
  "summary": "Malicious code in pipfontingv2 (PyPI)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "pipfontingv2",
        "purl": "pkg:pypi/pipfontingv2"
      },
      "versions": [
        "1.0.0"
      ]
    }
  ],
  "references": [
    {
      "type": "ARTICLE",
      "url": "https://medium.com/checkmarx-security/the-evolutionary-tale-of-a-persistent-python-threat-d19196d4ed15"
    },
    {
      "type": "ADVISORY",
      "url": "https://security.snyk.io/vuln/SNYK-PYTHON-PIPFONTINGV2-5932351"
    }
  ],
  "credits": [
    {
      "name": "ReversingLabs",
      "type": "FINDER",
      "contact": [
        "https://www.reversinglabs.com"
      ]
    }
  ],
  "database_specific": {
    "malicious-packages-origins": [
      {
        "source": "reversing-labs",
        "sha256": "27a782e6fe375dd8c410493e812dad2aa1c834b88eb51d05b9d9ce1f8f818258",
        "import_time": "2024-06-28T02:49:44.987061696Z",
        "id": "RLMA-2024-04307",
        "modified_time": "2024-06-25T13:38:31Z",
        "versions": [
          "1.0.0"
        ]
      },
      {
        "source": "reversing-labs",
        "sha256": "3da12b0e92bd5d07c3818a98ab03f32fcfae2d2784201b4887a63c3a1237a5d9",
        "import_time": "2024-10-24T00:59:38.454057449Z",
        "id": "RLUA-2024-08735",
        "modified_time": "2024-10-16T14:45:56Z"
      }
    ]
  }
}
